packages/espressocash_app/lib/features/outgoing_split_key_payments/screens/oskp_confirmation_screen.dart

```diff
@@ -85,11 +85,6 @@ class _TokenCreateLinkContent extends StatelessWidget {
   @override
   Widget build(BuildContext context) => Column(
         children: [
```

Comment:
Ok, since we change these modules anyway, and they will not work properly, let's remove them within this PR.